flake.nix

```diff
@@ -33,8 +34,35 @@
           inherit (pkgs.haskellPackages)
             cabal-fmt
             fourmolu;
+
+        };
+
+        packages.default = self'.packages.mood-tracker;
+
+      };
+      flake = {
+        nixosConfigurations.mood-tracker = nixpkgs.lib.nixosSystem {
+          system = "x86_64-linux";
+          modules = [
+            ./configuration.nix
+            ({ config, pkgs, ... }: {
+              imports = [ "${nixpkgs}/nixos/modules/virtualisation/openstack-config.nix" ];
+              environment.systemPackages = [ self.packages.x86_64-linux.mood-tracker ];
+            })
+          ];
+        };
+
+        deploy.nodes.mood-tracker = {
+          hostname = "root";
```

Comment:
Why is hostname "root"?

Reply:
I've altered this to "server" as we are using the machine primarily as a server.

Reply:
Take a look at the hostnames of some of our servers (ask Infra). Do any of them have such generic names like "server"? If not, what would be the hostname of the machine to which the mood-tracker is being deployed to and run as a web app?

Reply:
I'm waiting on John to give me some examples of this for now, its put down as "mood-tracker-server". However I will edit this later once I have that feedback.

Reply:
Now John has gotten back to me on this. I have changed it again to something more unique.